index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Adopt A Dog</title>
+    <!-- normalize linking -->
+    <link rel="stylesheet" href="./styles/normalize.css" />
+    <!-- stylesheet linking -->
+    <link rel="stylesheet" type="text/css" href="./styles/stylesheet.css">
+  </head>
+  <body>
+    <div class="wrapper" id="clearfix">
+      <header>
+        <div class="logo">
+          <h1><img src="./images/adoptadog-logo.svg" alt="adopt a dog logo"></h1>
+          <p>Seattle's premiere dog adoption agency</p>
+        </div>
+        <a href="#main" class="hidden">Skip to main content</a>
+        <div class="checkout">
+          <div>
+            <img src="./images/cart-icon.svg" alt="Shopping Cart Icon">
+            <p class="cost">$1234.56</p>
+          </div>
+          <button onclick="document.location='./checkout.html'">Checkout</button>
+        </div>
+      </header>
+      <nav class="menu">
+        <ul>
+          <li><a class="active" href="./index.html">Home</a></li>
+          <li>/</li>
+          <li><a href="./dogs.html">Dogs</a></li>
+          <li>/</li>
+          <li><a href="./blog.html">Blog</a></li>
+        </ul>
+      </nav>
+
+      <article class="main-content">
+        <div class="summary">
```

Comment:
Would suggest using flexbox here, specifically flex-wrap. This will allow your text to wrap under the image on smaller screen sizes. Your html here is great!